submissions/OlexiyDobroskok/task_js_dom/script.js

```diff
@@ -0,0 +1,146 @@
+    photo: "./img/Song of Susannah.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower VI: Song of Susannah",
+  },
+  {
+    id: "dt7",
+    title: "The Dark Tower VII: The Dark Tower",
+    quote:
+      "“Fault always lies in the same place, my fine babies: with him weak enough to lay blame.”",
+    about:
+      "The final volume sees gunslinger Roland on a roller-coaster mix of exhilarating triumph and aching loss in his unrelenting quest to reach the dark tower. Roland's band of pilgrims remains united, though scattered. Susannah-Mia has been carried off to New York to give birth, Terrified of what may happen.",
+    photo: "./img/The Dark Tower.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower VII: The Dark Tower",
+  },
+  {
+    id: "dt8",
+    title: "The Dark Tower: The Wind Through the Keyhole",
+    quote: "“Balls to your sorry ... Ka works and the world moves on. ”",
+    about:
+      "We join Roland and his ka-tet as a ferocious storm halts their progress along the Path of the Beam. As they shelter from the screaming wind and snapping trees, Roland tells them not just one strange tale, but two – and in doing so sheds fascinating light on his own troubled past.",
+    photo: "./img/The Wind through the Keyhole.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower: The Wind Through the Keyhole",
+  },
+];
+
+let main = document.querySelector(".content");
+let aside = document.createElement("aside");
+aside.className = "sidebar";
+main.prepend(aside);
+aside.prepend(...addButtons());
+
+function addButtons() {
+  const buttonArr = [];
+  for (let i = 0; i < 9; i++) {
+    let button = document.createElement("button");
+    button.className = "sidebar__button";
+    button.id = "dt" + [i];
+    button.innerHTML = darkTowerSeries[i].title;
+    buttonArr.push(button);
+  }
+  return buttonArr;
+}
+
+let section = document.createElement("section");
+section.className = "content__section";
+aside.after(section);
+let div = document.createElement("div");
+div.className = "textcontent__container";
+section.prepend(div);
+let h2 = document.createElement("h2");
+h2.className = "content__title";
+div.prepend(h2);
+let blockquote = document.createElement("blockquote");
+h2.after(blockquote);
+let paragraphQuote = document.createElement("p");
+paragraphQuote.className = "quote__text";
+blockquote.prepend(paragraphQuote);
+let paragraph = document.createElement("p");
+paragraph.className = "content__text";
+blockquote.after(paragraph);
+let img = document.createElement("img");
+img.className = "content__img";
+div.after(img);
+
+aside.onclick = function (event) {
+  let target = event.target;
+  if (target.tagName !== "BUTTON") return;
+  contentCreator(target);
+};
+
+function contentCreator(eventBtn) {
+  for (let i = 0; i < darkTowerSeries.length; i++) {
```

Comment:
I would recommend adding all variables which you create in this function.
Your function gets a parameter, it is good. Try to use a method of array `find` or `filter` to get needed item from array list.